sopel/plugins/capabilities.py

```diff
@@ -129,11 +228,40 @@ def request_available(
         :param bot: the bot instance used to send capability requests
         :param available_capabilities: available capabilities
 
-        This sends ``CAP REQ`` command for requests that can be made, i.e.
+        This sends ``CAP REQ`` commands for requests that can be made, i.e.
         all the requested capabilities (with or without prefix) must be
         available for Sopel to send the request.
 
-        Requests made are stored as requested, other are ignored.
+        Requests made are stored as requested; others are ignored::
+
+            >>> # manager.register => cap1 and cap2, not cap3
+            >>> manger.request_available(bot, ('cap1', 'cap3'))
+            >>> manger.is_requested(('cap1',))
+            True
+            >>> manger.is_requested(('cap2',))
+            False
+            >>> manger.is_requested(('cap3',))
```

Comment:
```suggestion
            >>> manager.request_available(bot, ('cap1', 'cap3'))
            >>> manager.is_requested(('cap1',))
            True
            >>> manager.is_requested(('cap2',))
            False
            >>> manager.is_requested(('cap3',))
```

No eating involved here!

(This joke works for both the English and French definitions of "manger"!)